README.md

```diff
@@ -1,2 +1,136 @@
+
+
+
+## 🚀 Live Demo & Video presentation <a name="live-demo"></a> <a name='video'></a>
+<br>
+
+- [Click me to view deployed project!](https://mohammadyaser.github.io/Module-1-Capstone/)
+
+- [Click me to view web page and code presentation](https://www.loom.com/share/a850d42c42e14f959c1e8f073896f76b)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+<br>
+
+A great project which can be used for almost anything, as it is made dynamically, so information can be updated or added through Javascript.
+
+To get a local copy up and running, follow these steps:
+<br>
+- Choose a directory
+- Open a Terminal
+- write git clone https://github.com/MohammadYaser/Module-1-Capstone.git
+- get into to directory "Capstone Project"
+- write: npm install
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+<br>
+
+
+👤 **Agneta Pupienyte**
+
+- GitHub: [@githubhandle](https://github.com/MohammadYaser)
+- Twitter: [@twitterhandle](https://twitter.com/Yaser_Safi19)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammad-yaser-safi-a12083270)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+<br>
+
+-**In the future there will be some of the transitions and animations added to the page, to have more inviting vibe for the user**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>Contributions, issues, and feature requests are welcome!
+<br>
+
+Feel free to check the [issues page](https://github.com/MohammadYaser/Module-1-Capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+<br>
+
+If you like my page please show your support and give this page a star!
+<br>
+<br>
+[![Star](https://img.shields.io/github/stars/MohammadYaser/Module-1-Capstone?style=social)](https://github.com/MohammadYaser/Module-1-Capstone)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+<br>
+
+#### **I would like to say a big thanks to original creator of this web page Cindy Shin, who designed it and allowed students to use her page as a template** ####
+
+<br>
+I would like to extend my heartfelt gratitude to Microverse for providing me with the opportunity to undertake this capstone project. Additionally, I am immensely thankful to the code reviewers who took the time to assess my work and provide valuable feedback, which greatly contributed to the refinement and success of the project. Your guidance and support have been invaluable throughout this solo endeavor, and I am truly grateful for your contributions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The license link leads to a 404 error page. I'll suggest you add a License file to your project and link it correctly here. :+1: :+1: 

- - Screenshot:
![Screenshot from 2023-07-07 11-16-06](https://github.com/MohammadYaser/Module-1-Capstone/assets/107632780/f221280d-c1dc-414a-a0c2-29cc8cd344b5)